src/containers/HubiiApiHoc/saga.js

```diff
@@ -98,28 +97,57 @@ export function* loadTransactions({ address }, endpoint) {
   }
 }
 
+export function* requestToken() {
+  while (true) { // eslint-disable-line no-constant-condition
```

Comment:
Just curious, what is the frequency the token would be changed?

Reply:
1 minute, there's a timer on ln 142